.github/workflows/docker-dist-tests.yml

```diff
@@ -28,4 +28,6 @@ jobs:
       nat_ip_auto: true
       tag_latest: ${{ github.ref_name == github.event.repository.default_branch || startsWith(github.ref, 'refs/tags/') }}
       tag_suffix: dist-tests
+      # continuous_tests_list: PeersTest HoldMyBeerTest
+      # continuous_tests_duration: 3h
```

Comment:
Leftover?

Reply:
This is basically how we can enable Continuous Tests run on commits and they were disabled.

Uncommented the line - let's run them and check how it works :)